packages/stream_video/lib/src/call/session/call_session.dart

```diff
@@ -602,6 +604,23 @@ class CallSession extends Disposable {
         raw: rtcRawStats,
       ),
     );
+
+    sfuClient.sendStats(
```

Comment:
Is is ok that we will send publisherStats and subsciberStats separately? Shouldn't we wait to have both and send it together? It also means that we won't respect `reportingIntervalMs` fully since we will send two reports every interval.